.circleci/config.yml

```diff
@@ -174,6 +174,15 @@ jobs:
     steps:
       - checkout
 
+      - restore_cache:
+          key: bundle-v1-{{ checksum "ios/Gemfile.lock" }}-{{ arch }}
+
+      - restore_cache:
+          key: yarn-v1-{{ checksum "yarn.lock" }}-{{ arch }}
```

Comment:
why `arch`?

Reply:
because this is the way it's suggested in docs

Reply:
https://docs.fastlane.tools/best-practices/continuous-integration/circle-ci/

Reply:
I guess we may not need it.
We should try to use same cache for all jobs, as all of them uses `node_modules`.